src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        Ball ball = new Ball();
+        int value = random.nextInt(100);
```

Comment:
make 100 a class-level constant (use CONSTANT_CASE for naming)

Reply:
not fixed